src/types.ts

```diff
@@ -0,0 +1,2 @@
+export type URI = string;
```

Comment:
:+1: for those types 🥇 

I have only mixed feelings is it `URI` maybe it should be `ProblemDetailsTypeURI` or something.